airbyte-oauth/src/main/java/io/airbyte/oauth/flows/facebook/FacebookOAuthFlow.java

```diff
@@ -70,19 +68,19 @@ protected String getAccessTokenUrl() {
   }
 
   @Override
-  protected Map<String, Object> completeOAuthFlow(final String clientId,
-                                                  final String clientSecret,
-                                                  final String authCode,
-                                                  final String redirectUrl,
-                                                  final JsonNode oAuthParamConfig)
+  protected Map<String, Object> internalCompleteOAuthFlow(final String clientId,
+                                                          final String clientSecret,
+                                                          final String authCode,
+                                                          final String redirectUrl,
+                                                          final JsonNode oAuthParamConfig)
       throws IOException {
     // Access tokens generated via web login are short-lived tokens
     // they arre valid for 1 hour and need to be exchanged for long-lived access token
     // https://developers.facebook.com/docs/facebook-login/access-tokens (Short-Term Tokens and
     // https://developers.facebook.com/docs/instagram-basic-display-api/overview#short-lived-access-tokens
     // Long-Term Tokens section)
 
-    final Map<String, Object> data = super.completeOAuthFlow(clientId, clientSecret, authCode, redirectUrl, oAuthParamConfig);
+    final Map<String, Object> data = super.internalCompleteOAuthFlow(clientId, clientSecret, authCode, redirectUrl, oAuthParamConfig);
```

Comment:
This is a bit smelly because we are calling something called `internal` from outside that class. Why aren't we calling `completeOauthflow` instead? I think if it is meant to be called from outside the class then it is not `internal` 😅